src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx

```diff
@@ -107,14 +107,14 @@ class EstimateYourBenefitsForm extends React.Component {
       this.displayExtensionBeneficiaryZipcode() &&
       (beneficiaryZIPError || beneficiaryZIP.length !== 5)
     ) {
-      this.toggleLearningFormatAndSchedule(true);
-      setTimeout(() => {
-        scroller.scrollTo('beneficiary-zip-question', getScrollOptions());
-        focusElement('input[name=beneficiaryZIPCode]');
-      }, 1);
+      await this.toggleLearningFormatAndSchedule(true);
+      scroller.scrollTo('beneficiary-zip-question', getScrollOptions());
+      focusElement('input[name=beneficiaryZIPCode]');
     } else {
       this.setState({ inputUpdated: false });
-      this.props.updateEstimatedBenefits();
+      await this.props.updateEstimatedBenefits();
+      scroller.scrollTo(accordionButtonId, getScrollOptions());
+      focusElement(`#${accordionButtonId}`);
```

Comment:
Why not focus on `this.id = `${createId(props.button)}-accordion`;` from `AccordionItem` ?

Reply:
When you resolve merge conflicts keep incoming property name and usage and do
``${createId(childSectionName)}-button`;` or ``${createId(childSectionName)}-accordion`;` to necessary id

Reply:
I didn't realize that was there. I thought we were using the id being passed in props. But as it turns out, that prop isn't even being used. I made the change and removed the prop.